Misc fixes around ElevatedView (#3557)
unoplatform/uno: doc/articles/features/ElevatedView.md

# ElevatedView
On many design language like _Material Design_, there's a notion of elevation where a portion of the UI should be presented as been _elevated_ over the rest of the content.

In this case, the UWP proposed solution can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar result on all platforms, providing an elevated result.

This control is very useful to create cards with both rounded corners and elevated effect - which could be challenging to produce on some platforms.

## How to use the `ElevatedView`

> 

Reviewer: First step is to have the nuget reference to Uno or Uno.UI.Toolkit, it's often forgotten on UWP.

> Fist you need to add the `toolkit` namespace in your XAML file:

Reviewer: ```suggestion
First you need to add the `toolkit` namespace in your XAML file:
```

# ElevatedView

> On many design languages like _Material Design_, there's a notion of elevation where a portion of the UI should be presented has been _elevated_ over the rest of the content.

Reviewer 2: ```suggestion
In many design languages (like [_Material Design_](https://material.io/design)), there's a notion of 'elevation' where a portion of the UI should be presented as being _elevated_ over the rest of the content.
```

> In this case, the UWP proposed solution can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar result on all platforms, providing an elevated result.

Reviewer 2: ```suggestion
In this case, UWP's native elevation API can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar elevated effect on all platforms (UWP, Android, iOS, WebAssembly, and macOS).
```

> This control is very useful to create cards with both rounded corners and elevated effect - which could be challenging to produce on some platforms.

Reviewer 2: ```suggestion
This control is very useful to create cards with both rounded corners and an elevated effect - which could otherwise be challenging to produce on some platforms.
```

## How to use the `ElevatedView`

First you need to add the `toolkit` namespace in your XAML file:

```
xmlns:toolkit="using:Uno.UI.Toolkit"
```

After that, use the `ElevatedView` to host the content you need to be elevated:
``` xml
<StackPanel Orientation="Horizontal" Spacing="20">

	<Button>Non-Elevated Button</Button>

	<toolkit:ElevatedView Elevation="10">
		<Button>Elevated Button</Button>
	</toolkit:ElevatedView>

</StackPanel>
```

Will produce the following result:

![ElevatedView sample](../Assets/features/elevatedview/elevatedview-sample.png)

> > **ATTENTION FOR UWP**: When there is an error seeing the `<toolkit:ElevatedView>` on UWP, the common mistake is to forget to include the `Uno.UI` package for all platforms, including UWP. On this platform, the only available component is the Toolkit.

Reviewer 2: ```suggestion
> **ATTENTION FOR UWP**: When there is an error seeing the `<toolkit:ElevatedView>` on UWP, the common mistake is to forget to include the `Uno.UI` package for all platforms, including UWP. On UWP, the only component that the `Uno.UI` package adds is the Toolkit.
```

## Settings

You can set the following properties:

> * `Elevation`: numeric number representing the level of the elevation effect. Typical values are between 5 and 30. Default to `0` - no elevation.

Reviewer 2: ```suggestion
* `Elevation`: numeric number representing the level of the elevation effect. Typical values are between 5 and 30. The default is `0` - no elevation.
```

Also - not really relevant here, but that's a weird default.

> * `ShadowColor`: By default the casted shadow will be `Black`, but you can set any other value. You can reduce the shadow effect by using the alpha channel [except Android]. On Android, the shadow color can only be changed since Android Pie (API 28+). Default to `Black` with alpha channel at 25%.

Reviewer 2: ```suggestion
* `ShadowColor`: By default the shadow will be `Black`, but you can set any other value. You can reduce the shadow effect by using the alpha channel [except Android]. On Android, the shadow color can only be changed since Android Pie (API 28+). The default is `Black` with alpha channel at 25%.
```

> * `Background`: Default to `Transparent`. Setting `null` will remove the shadow effect.

Reviewer 2: ```suggestion
* `Background`: The default is `Transparent`. Setting `null` will remove the shadow effect.
```

> * `CornerRadius`: Use it to create rounded corner effects. The shadow should follow them.

Reviewer 2: ```suggestion
* `CornerRadius`: Use it to create rounded corner effects. The shadow will follow them.
```

## Particularities

> * Make sure to _give room_ for the shadow in the layout.  Some platforms like macOS could easily clip the shadow. For the same reason, avoid wrapping the `<toolkit:ElevatedView>` directly in a `<ScrollViewer> ` because it's designed to clip its content.

Reviewer 2: ```suggestion
* Make sure to _give room_ for the shadow in the layout (eg by setting a `Margin` on the `ElevatedView`).  Some platforms like macOS may clip the shadow otherwise. For the same reason, avoid wrapping the `<toolkit:ElevatedView>` directly in a `<ScrollViewer> ` because it's designed to clip its content.
```
